src/cloudflare/internal/workflows-api.ts

```diff
@@ -40,41 +40,39 @@ async function callFetcher<T>(
 }
 
 class InstanceImpl implements WorkflowInstance {
-  // TODO(soon): Can we use the # syntax here?
-  // eslint-disable-next-line no-restricted-syntax
-  private readonly fetcher: Fetcher;
```

Comment:
(a bunch of private statements are being dropped, I assume that's intentional)

Reply:
It was intentional but it makes review harder. I'll open a new PR for this.